Signal/src/call/PeerConnectionClient.swift

```diff
@@ -115,7 +115,14 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
 
     private var videoSender: RTCRtpSender?
     private var localVideoTrack: RTCVideoTrack?
-    private weak var remoteVideoTrack: RTCVideoTrack?
+    // RTCVideoTrack is fragile and prone to throwing exceptions and/or
```

Comment:
Yes, I've been wondering if I should do something similar for these properties.  I've just never seen an issue with them during any of my (at this point insanely rigorous) testing.